test/e2e/network_test.go

```diff
@@ -297,6 +298,23 @@ var _ = Describe("Podman network", func() {
 		Expect(rmAll).Should(ExitCleanly())
 	})
 
+	It("podman run container host interface name", func() {
+		Skip("We need to update netavark first.")
+
+		ctrName := "testCtr"
+		vethName := "my_veth"
```

Comment:
please do not use static names, that might cause conflicts on the host.
Append a random suffix, i.e. `"my_veth" + stringid.GenerateRandomID()[:8]`